instant-gram/AppDelegate.m

```diff
@@ -23,6 +23,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
         configuration.server = @"https://instant-gram.herokuapp.com/parse";
     }];
     
+    [Parse initializeWithConfiguration:config];
```

Comment:
Careful of adding extra stuff to commits. Run git diff to see the exact changes. We should go over more about stages vs unstaged changes.